scripts/initialize.ts

```diff
@@ -244,6 +264,7 @@ export const initialize = async () => {
       message: 'Would you like any of the following (optional)?',
       options: [
         { label: 'Pre-commit hook with Husky', value: 'precommit-hooks' },
```

Comment:
Can you change the labels here to:

- Husky pre-commit hook
- Lefthook pre-commit hook

Just a bit shorter.